transport-native-epoll/src/main/c/netty_unix_socket.c

```diff
@@ -708,6 +730,12 @@ static JNINativeMethod* createDynamicMethodsTable(const char* packagePrefix) {
     dynamicMethod->signature = netty_unix_util_prepend("(IJII)L", dynamicTypeName);
     dynamicMethod->fnPtr = (void *) netty_unix_socket_recvFromAddress;
     free(dynamicTypeName);
+    ++dynamicMethod;
+    dynamicTypeName = netty_unix_util_prepend(packagePrefix, "io/netty/channel/unix/PeerCredentials;");
+    dynamicMethod->name = "peerCredentials";
+    dynamicMethod->signature = netty_unix_util_prepend("(ILio/netty/channel/unix/PeerCredentials;)I", "");
```

Comment:
is the intention to use `dynamicTypeName` here? It doesn't look like the signature is quite correct. Whenever a netty type is used you should make sure to use the `netty_unix_util_prepend` method. Here you directly use `io/netty/channel/unix/PeerCredentials` which will not be correct if the Netty library is shaded.